modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -184,6 +183,26 @@ model EventSubscription {
 }
 
 model OnchainTransaction {
+  // Response fields
+  id String @id @default(uuid())
+
+  // Channel fields
+  status         String  @default("submitted") // no enums
+  reason         String // no enums
+  error          String?
+  channelAddress String
+  channel        Channel @relation(fields: [channelAddress], references: [channelAddress])
+
+  createdAt DateTime @default(now())
+
+  confirmedReceipt OnchainTransactionReceipt? @relation(fields: [confirmedReceiptHash], references: [transactionHash])
+  confirmedReceiptHash String?
+  onchainTransactionAttempts OnchainTransactionAttempt[] @relation
```

Comment:
so the final gas price for this tx will be the gas price belonging to the last tx in this list?

Reply:
Also - I believe `@relation` requires a method call structure and needs a `references` argument.

So... `@relation(name: "OnchainTransactionAttempt", references: [id]` ?

Reply:
Ignore above

Remove `@relation` here, should only be on the many side of the relationship